api/v1beta1/aerospikecluster_types.go

```diff
@@ -390,17 +390,40 @@ const (
 	AerospikeVolumeInitMethodDeleteFiles AerospikeVolumeInitMethod = "deleteFiles"
 )
 
+// AerospikeVolumeWipeMethod specifies how block volumes should be wiped.
+// +kubebuilder:validation:Enum=dd;blkdiscard;deleteFiles
+// +k8s:openapi-gen=true
+type AerospikeVolumeWipeMethod string
+
+const (
```

Comment:
I think you can reuse the init method enum instead of creating a new wipe method enum.

Reply:
If we use init enum then we have to add a check to invalidate the `AerospikeVolumeInitMethodNone` method

Reply:
Yes ... None is not allowed